### ISSUE TITLE
German localization

### ISSUE THREAD

**Issue by bskhsup013** [root3nl/SupportApp#199]
Hello, we use this tool on all Macs in our company. We would like to change the uptime text, as it is here recommended to restart every day. If I set the number to zero or one via Jamf, then unfortunately the German is wrong. Can we change this somehow, or would you have to do it? 

It should say : Dein Administrator empfiehlt Dir einen Neustart täglich durchzuführen

Thank you for this great app

**Comment by Tommyfare** [comment]
You can add the property "Update Text" and write whatever you want. 
<img width="1154" alt="Screenshot 2024-11-07 at 12 03 58 PM" src="https://github.com/user-attachments/assets/16ae50f1-2a7c-41d8-8c78-66214d2ce278">

**Comment by jordywitteman** [comment]
Okay thanks for flagging this! So if I understand correctly this is only an issue when the restart is recommended at a daily basis and not every more than 1 days? In other words, setting `UptimeDaysLimit` to 1 shows an incorrect sentence and correct when set to >1? Which sentence of the three provided here is the best?

**Comment by bskhsup013** [comment]
Hi,

that's exactly how it is. If a 1 or a 0 in UptimeDaysLimit it says :

Dein Administrator empfiehlt Dir einen Neustart alle Tag.

It should be :

Dein Administrator empfiehlt Dir einen Neustart jeden Tag

Or better :

Dein Administrator empfiehlt Dir einen täglichen Neustart.

When its set to >1 it is correct.

Greetings

Von: Jordy Witteman ***@***.***>
Datum: Mittwoch, 13. November 2024 um 23:05
An: root3nl/SupportApp ***@***.***>
Cc: Thrun, Adam - Externer Mitarbeiter ***@***.***>, Author ***@***.***>
Betreff: [EXT] Re: [root3nl/SupportApp] German localization (Issue #199)
Sie erhalten nicht häufig E-Mails von ***@***.*** Erfahren Sie, warum dies wichtig ist<https://aka.ms/LearnAboutSenderIdentification>

Okay thanks for flagging this! So if I understand correctly this is only an issue when the restart is recommended at a daily basis and not every more than 1 days? In other words, setting UptimeDaysLimit to 1 shows an incorrect sentence and correct when set to >1? Which sentence of the three provided here is the best?

**Comment by jordywitteman** [comment]
@bskhsup013 thanks for clarifying! In Slack other people agree on this translation: **Dein Administrator empfiehlt Dir, täglich einen Neustart durchzuführen.**

For now that's what I will add unless that doesn't make sense. We can ship this in a bug fix release 2.6.1

**Comment by bskhsup013** [comment]
Yes , it doesn’t make any sense. I know , but that’s what they want.

Many thanks 😊

Von: Jordy Witteman ***@***.***>
Datum: Donnerstag, 14. November 2024 um 12:02
An: root3nl/SupportApp ***@***.***>
Cc: Thrun, Adam - Externer Mitarbeiter ***@***.***>, Mention ***@***.***>
Betreff: [EXT] Re: [root3nl/SupportApp] German localization (Issue #199)
Sie erhalten nicht häufig E-Mails von ***@***.*** Erfahren Sie, warum dies wichtig ist<https://aka.ms/LearnAboutSenderIdentification>

@bskhsup013<https://github.com/bskhsup013> thanks for clarifying! In Slack other people agree on this translation: Dein Administrator empfiehlt Dir, täglich einen Neustart durchzuführen.

For now that's what I will add unless that doesn't make sense. We can ship this in a bug fix release 2.6.1

**Comment by cubandave** [comment]
Wouldn't it be better to use the Sie form for all the notifications?

**Comment by dernerl** [comment]
I would find the “you” form much better!
